Helper/Data.php

```diff
@@ -167,6 +184,37 @@ public function getBuyerAmount()
         return $response;
     }
 
+    /**
+     * Update Buyer Id
+     *
+     * @param $buyerId
+     */
+    public function updateBuyerId($buyerId)
+    {
+        $customer = $this->customer->load($this->customerSession->getCustomer()->getId());
+        $customerData = $customer->getDataModel();
+        $customerData->setCustomAttribute('buyer_id', $buyerId);
+        $customer->updateData($customerData);
+        $customerResource = $this->customerFactory->create();
+        $customerResource->saveAttribute($customer, 'buyer_id');
+    }
+
+    /**
+     * GetBuyerId
+     *
+     * @return mixed|string
+     * @throws \Magento\Framework\Exception\LocalizedException
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
+     */
+    public function getBuyerId()
```

Comment:
Let's move all buyer related functionality to more appropriate class (like BalanceBuyer class)

Reply:
Consider renaming to `getCustomerBalanceBuyerId`